contrib/linearize/linearize-data.py

```diff
@@ -102,6 +103,13 @@ def __init__(self, settings, blkindex, blkmap):
         self.blkmap = blkmap
 
         self.inFn = 0
+        fnPattern = os.path.join(self.settings['input'], "blk[0-9][0-9][0-9][0-9][0-9].dat")
```

Comment:
Could add a comment here explaining the following lines.

Reply:
Added comment. Please feel free to modify at the discretion.

Reply:
would make sense to factor this out to a function, `getFirstBlockFileId(path)` or such